src/main/java/io/appium/java_client/touch/offset/ElementOption.java

```diff
@@ -85,8 +85,8 @@ public ElementOption withElement(WebElement element) {
         checkNotNull(element);
         checkArgument(true, "Element should be an instance of the class which "
                 + "extends org.openqa.selenium.remote.RemoteWebElement",
-                (RemoteWebElement.class.isAssignableFrom(element.getClass())));
-        elementId = RemoteWebElement.class.cast(element).getId();
+                (HasIdentity.class.isAssignableFrom(element.getClass())));
```

Comment:
why not to use instanceof?

Reply:
I don't know. I just tried to make a minimal change. You never know what can go wrong. :)

Reply:
Nothing is ideal. Lets do it properly

Reply:
done